server/webhook_worker.go

```diff
@@ -13,12 +13,22 @@ func (ww webhookWorker) work() {
 	for bb := range ww.workQueue {
 		wh, err := ParseWebhook(bb)
 		if err != nil {
-			ww.p.errorf("webhookWorker id: %d, error parsing webhook, err: %v", ww.id, err)
+			// Don't log an error if we're just ignoring the webhook.
+			if err != ErrWebhookIgnored {
+				ww.p.errorf("webhookWorker id: %d, error parsing webhook, err: %v", ww.id, err)
+			}
+			continue
+		}
+
+		_, _, err = wh.PostNotifications(ww.p)
+		if err != nil {
+			ww.p.errorf("WebhookWorker id: %d, error posting notifications, err: %v", ww.id, err)
 		}
 
 		channelIds, err := ww.p.getChannelsSubscribed(wh.(*webhook))
 		if err != nil {
 			ww.p.errorf("webhookWorker id: %d, error getting channel's subscribed, err: %v", ww.id, err)
+			continue
```

Comment:
I know this isn't completely necessary, but it declares our intention: if there's an error, the rest is skipped.